src/cliActions.js

```diff
@@ -119,13 +119,12 @@ const signingKeyChangePW = (
   });
 
 /**
- * @desc Streamming Dapp
+ * @desc Check if signing key exist and decrypt it
  * @param  {string} pw password of signing key
  * @param  {string} signingKeyFile signing key file path
- * @param  {bool} devMode use dev mode to Streamming Dapp
  * @return {Promise<Promise>}
  */
-const dappStreaming = ({ pw }, signingKeyFile, devMode) =>
+const processSigningKey = ({ pw }, signingKeyFile) =>
```

Comment:
this function is not supposed to be in this file. This is a utility. I would also rename this function `processSigningKey` is not expressive enought

Reply:
done